### ISSUE TITLE
Export of Polygons does not work properly

### ISSUE THREAD

**Issue by ktrinko** [NationalSecurityAgency/qgis-kmltools-plugin#32]
Exporting Polygonal-Files from QGIS to KMZ does not work for me. (Plugin-Version: [3.1.30] , QGIS: 3.32.2 , Win 10 64 bit )
I enclose an example:
![QGIS](https://github.com/NationalSecurityAgency/qgis-kmltools-plugin/assets/110204960/1d493b07-df93-48f9-a25c-72c6f2ea2bbb)
![GoogleEarth](https://github.com/NationalSecurityAgency/qgis-kmltools-plugin/assets/110204960/873c700b-1ecf-40bd-9295-b334cfa8ec21)
[Files.zip](https://github.com/NationalSecurityAgency/qgis-kmltools-plugin/files/12566369/Files.zip)

**Comment by hamiltoncj** [comment]
That is really strange. I will see what can be done to fix it.

**Comment by ktrinko** [comment]
Dear Sir,

thanks for your kind support!

Am Di., 12. Sept. 2023 um 18:46 Uhr schrieb C Hamilton <
***@***.***>:

**Comment by hamiltoncj** [comment]
I have looked carefully at the generated output of the KML and cannot find anything wrong. I have found that if I simplify/reduce the number of polygon points then it renders properly in Google Earth. Your polygon features are very complex and I am beginning to think that the problem is not with the KML output, but with Google Earth's ability to render polygons of that complexity. Actually, in doing a search on KML limits, a polygon cannot have over 30,000 vertices. I think these exceed that.
